### PR TITLE
Add intro git numbers to attendance

### PR DESCRIPTION
**Pull request by mbonsma**
The number of attendees is +-1, I estimated based on who I remember being there.

### CODE REVIEW

File: Attendance.csv [hunk 1]
```diff
@@ -1,2 +1,2 @@
 Date,Session,Attendance,Notes,
-2017-01-11,Intro Git,x,,
+2017-01-11,Intro Git,9,3 physics 2 biomed engineering 1 undergrad math 1 psychology and Luke,
```
Comment (QuLogic) [inline Attendance.csv]: What about the lady who came late?

Comment (mbonsma) [inline Attendance.csv]: Tammy? I'll add her, good call.

Comment (mbonsma) [inline Attendance.csv]: The total is unchanged though, I think I counted her but didn't put her in the description.

Comment (QuLogic) [inline Attendance.csv]: Ah, I forgot to add them together to see.

**Pull request merged**